src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/ReadOnlyTensorSpan.cs

```diff
@@ -554,13 +556,14 @@ public void CopyTo(scoped TensorSpan<T> destination)
         }
 
         /// <summary>
-        /// Copies the contents of this read-only span into destination span. If the source
-        /// and destinations overlap, this method behaves as if the original values in
-        /// a temporary location before the destination is overwritten.
+        /// Copies the contents of this read-only span into destination span.
         /// </summary>
-        /// <returns>If the destination span is shorter than the source span, this method
-        /// return false and no data is written to the destination.</returns>
+        /// <returns><see langword="true"/> if the copy succeeded; <see langword="false"/> if the destination span is shorter than the source span and no data is written to the destination.</returns>
         /// <param name="destination">The span to copy items into.</param>
+        /// <remarks>
+        /// If the source and destinations overlap, this method behaves as if
+        /// the original values are in a temporary location before the destination is overwritten.
+        /// </remarks
```

Comment:
Build failure source:

```suggestion
        /// </remarks>
```